ui/app/src/state/actions/content.ts

```diff
@@ -80,3 +80,8 @@ export const pasteItemWithPolicyValidation = /*#__PURE__*/ createAction<{ path:
 // endregion
 
 export const unlockItem = /*#__PURE__*/ createAction<{ path: string; notify?: boolean }>('UNLOCK_ITEM');
+
+// To unlock a item the item.lockOwner should be empty or should be equals to the current user, this action check if can be unlocked
```

Comment:
Why would you issue an unlock call for an item that's already unlocked?
```suggestion
// This action's semantic is to unlock the item only if the lock owner is the current user.
```